uvm/servlets/setup/app/view/Main.js

```diff
@@ -171,9 +171,19 @@ Ext.define('Ung.Setup.Main', {
                 if(Util.setRpcJsonrpc("admin") == true){
                     me.resetWizardContinue();
                 }else{
-                    Util.authenticate("passwd", function () {
-                        me.resetWizardContinue();
-                    });
+                    var msg = Ext.create('Ext.window.MessageBox');
```

Comment:
Unfortunately, this means authentication will always be required.

I think what we should do instead is to keep the existing Util.authenticate functionality (and still calling with the default "passwd") but edit the current "failure" behavior to do the prompting you have here. 

It may be a kind of ugly looking loop, but this is not  a common occurance anyway.

Reply:
@cblaise-untangle  Added changes as suggested. kept existing functionality and still calling with the default "passwd")  and edited current behavior to prompt in case its non admin password.